packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -98,18 +110,19 @@ export default class InsertTableView extends View {
 
 				click: bind.to( () => {
 					this.fire( 'execute' );
+				} ),
+
+				keydown: bind.to( evt => {
+					if ( evt.key === 'Enter' ) {
+						this.fire( 'execute' );
```

Comment:
No sure which piece of logic is responsible for this but clicking a toolbar button followed by enter creates a 2x2 table (should be 1x1, I guess)
![image](https://user-images.githubusercontent.com/95343030/183961487-6516670f-163b-4c17-9c0f-585cf9651e19.png)

Reply:
This was happening on the second (and next) opening dropdown. It was caused by resetting selected rows/cols on dropdown opening in table.ui. I removed this code as now on opening dropdown we always focus the first tile so there is no need of resetting it somewhere else.